packages/marko-web-theme-monorail/components/content/body-with-injection.marko

```diff
@@ -0,0 +1,77 @@
+import { getAsArray } from "@parameter1/base-cms-object-path";
+import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+import MarkoWebContentBody from "@parameter1/base-cms-marko-web/components/element/content/body";
+import GAMDefineDisplayAd from "@parameter1/base-cms-marko-web-theme-monorail/components/gam/define-display-ad";
+
+$ const { global: $global } = out;
+$ const { GAM } = $global;
+$ const { content } = input;
+$ const preventHTMLInjection = getAsArray(content, "labels").some( l => ["Sponsored", "Product Spotlight"].indexOf(l) >= 0) || input.preventHTMLInjection;
+$ const blockName = defaultValue(input.blockName, 'page-contents');
+$ const aliases = getAsArray(input, 'aliases');
+$ const bodyId = `content-body-${content.id}`;
+
+<!-- Desktop Inline Ad props -->
+$ const desktopAdCounts = defaultValue(input.desktopAdCounts, [1000, 2750, 4500, 6250, 8000, 9750, 11500, 13250, 15000, 16750, 18500]);
+$ const desktopAdName = defaultValue(input.desktopAdName, "inline-content-desktop");
+$ const desktopAdModifiers = defaultValue(input.desktopAdModifiers, ["margin-auto-x",  "inline-content", "inline-content-desktop"]);
+$ const desktopAdDataPropToPrevetDups = defaultValue(input.desktopAdDataPropToPrevetDups, "gamTemplateName");
```

Comment:
@B77Mills - replace `PrevetDups` with `PreventDupes` in all files.  Missing the letter "n" in Prevent, and "e" in Dupes.